src/qt/bitcoinstrings.cpp

```diff
@@ -189,7 +183,7 @@ QT_TRANSLATE_NOOP("bitcoin-core", "Invalid netmask specified in -whitelist: '%s'
 QT_TRANSLATE_NOOP("bitcoin-core", "Loading P2P addresses…"),
 QT_TRANSLATE_NOOP("bitcoin-core", "Loading banlist…"),
 QT_TRANSLATE_NOOP("bitcoin-core", "Loading block index…"),
-QT_TRANSLATE_NOOP("bitcoin-core", "Loading wallet…"),
+QT_TRANSLATE_NOOP("bitcoin-core", "Loading wallet..."),
```

Comment:
Strange, why is this changing back

Reply:
Looks like bitcoin/bitcoin#20773 accidentally reverted this

Reply:
yeah...

don't throw a stone at me, but a linter could help

Reply:
Please no…

Reply:
Fixed in the recent push.